ere-guests/Cargo.toml

```diff
@@ -170,14 +170,16 @@ revm-bytecode = { version = "7.0.2", default-features = false }
 
 # misc
 bincode = "1.3"
+bincode_v2 = { package = "bincode", version = "2.0.1", default-features = false }
```

Comment:
To avoid this PR becoming a huge PR, I didn't try to update all bincode to v2, so here we name it like this